services/time/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "exonum-time"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
Same as with configuration, but for a different reason, which I shouldn't disclose - we *know* there is an issue with this service in the current form.